dora/core/server/worker/src/main/java/alluxio/worker/http/HttpServerHandler.java

```diff
@@ -186,4 +187,10 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) {
     cause.printStackTrace();
     ctx.close();
   }
+
+  @Override
+  public void handlerRemoved(ChannelHandlerContext ctx) throws Exception {
+    super.handlerRemoved(ctx);
+    mFileSystem.close();
```

Comment:
Closing `FileSystem` does not close the `FileSystemContext` it was created with.

Reply:
Added `mFileSystemContext.close();`.